addons/cookoff/functions/fnc_cookOff.sqf

```diff
@@ -122,5 +124,8 @@ if (_smokeDelayEnabled) then {
                 _vehicle setVariable [QGVAR(nextExplosiveDetonation), random 60];
             };
         };
+        
+        [_vehicle, [((_tiPars select 0) + (_intensity * 0.01))/1.005, ((_tiPars select 1) + (_intensity * 0.004))/1.002, ((_tiPars select 2) + (_intensity * 0.01)/1.005)]] call CBA_fnc_globalEvent;
```

Comment:
Seems wrong - no event name.
Also, please add some line breaks. This is almost 200 chars long...